packages/css/src/components/menu/menu.scss

```diff
@@ -35,7 +65,7 @@
 .ams-menu__link {
   align-items: flex-start;
   color: var(--ams-menu-link-color);
-  display: flex;
+  display: inline-flex;
```

Comment:
Links should only limit their width (and hit area) in a narrow window, but stretch (e.g. through `display: flex`) in the narrow Menu for a wide window.